README.md

```diff
@@ -0,0 +1,117 @@
+   make sure you verify the downloaded file's SHA-256 with the one at
+   the link under the anchor text 'SHA-256' ajacent to prior link.
+2. Write the image to a USB drive you are happy wiping contents from.
+   On Linux or OS X you can do this with the command:
+   `sudo dd bs=4M if=path/to/nixos-minimal-15.09.Y.Z-x86_64-linux.iso of=/dev/sdX`
+   where `X` is the letter of the device on the system you are creating
+   the bootable USB installer from.
+3. Ensure your Yoga 900 is in a powered off state.
+4. On the right hand side of your Yoga 900 there is a small penpoint button
+   between two externally obtruding buttons. Use a pencil or stylus tip to
+   press the round penpoint button and wait for the BIOS menu, select an
+   option titled 'Boot menu' (or similar, third one down on mine).
+5. Select the USB drive to boot from in the boot media menu screen.
+6. Once the bootable USB loads the root command prompt run the following
+   commands:
+
+```bash
+$ fdisk -l /dev/sda
+Disk /dev/sda: 477 GiB, 512110190592 bytes, 1000215216 sectors
+Units: sectors of 1 * 512 = 512 bytes
+Sector size (logical/physical): 512 bytes / 512 bytes
+I/O size (minimum/optimal): 512 bytes / 512 bytes
+Disklabel type: gpt
+Disk identifier: ???
+
+Device         Start        End   Sectors   Size Type
+/dev/sda1       2048     206847    204800   100M EFI System
+/dev/sda2     206848     239615     32768    16M Microsoft reserved
+/dev/sda3     239616  997105663 996866048 475.4G Windows ???
+/dev/sda4  997105664  998641663   1536000   750M Windows recovery environment
+/dev/sda5  998641664 1000214527   1572864   768M Lenovo boot partition
+```
+
+   You will see that the third partition (`/dev/sda3`) has Windows data on it.
+   Assuming you want to erase that partition (if you don't, this repository is
+   not for you and you should abort immediately) then you should delete the
+   partition, create a new Linux filesystem partition across the same sectors
+   and write the changes to the partition table.
+
+   To do this:
+
+```bash
+TODO: parted commands here
+```
+
+7. Encrypt `/dev/sda3` and mount necessary partitions to prepare for
+   installation like so:
+
+```bash
+$ cryptsetup luksFormat /dev/sda3
+$ cryptsetup luksOpen /dev/sda3 nixosroot
+$ mkfs.ext4 /dev/mapper/nixosroot
+$ mount /dev/mapper/nixosroot /mnt
+$ mount /dev/sda1 /mnt/boot
+```
+
+8. Setup `/mnt/etc/nixos` dir as in `etc/nixos` dir in  repo for Yoga 900
+   hardware. This has been tested on my Yoga 900-13ISK. See
+   [etc/nixos](etc/nixos). You can just edit [vars.nix](etc/nixos/vars.nix)
+   with your overrides. I have not included my slim/xmonad configuration
+   here, so you should add your own X customizations.
+
+9. Update the root Nix channel for `nixos` with the latest 16.09pre release.
+   I used the following:
+
+```bash
+$ nix-channel -add http://nixos.org/releases/nixos/unstable/nixos-16.09pre89269.8512747 nixos
```

Comment:
I did: `$ nix-channel --add https://nixos.org/channels/nixos-unstable nixos` because I am terrible (but may live better in the docs)